roles/ceilometer-control/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: restart ceilometer services
+  service: name={{ item }} state=restarted must_exist=false
+  when: restart|default('True')
+  with_items:
+    - ceilometer-api
+    - ceilometer-collector
+    - ceilometer-agent-notification
+    - ceilometer-agent-central
+    - ceilometer-alarm-evaluator
+    - ceilometer-alarm-notifier
```

Comment:
Would it make more sense to put the handlers in the ceilometer-common role? That way all the handlers can live in one place but be called by anything that lists ceilometer-common as a requirement.